README.md

```diff
@@ -40,6 +40,8 @@ Plugin should work with default settings. If not:
 
 5.  `always use elixirc` option - leave it disabled, unless `mix compile` is too slow.
 
+6.  `mix env` option - leave it at the default (dev), unless you want to lint in a specific environment. This can also be useful if changes made in Atom are not being picked up in IEx.
```

Comment:
How about this?:

Allows changing the build environment for lint runs. If using IEx at the same time as Atom this needs to be changed to allow IEx to pick up code changes.